examples/Microsoft.Spark.CSharp.Examples/MachineLearning/SentimentStream/Resources/yelptrain.csv

```diff
@@ -0,0 +1,500 @@
+ReviewText,Sentiment
```

Comment:
is  this the same file as the other sentiment example?  can we just include this once and not multiple times ?

Reply:
That's a good point, it seems unnecessary to upload twice. I'll just address the streaming ML sample's readme to reflect the change of where to find the file.

Reply:
I also realize that it uses the same MLModel, so I'll remove the Resources folder from here and just point to the one in the batch sentiment folder.